src/test/ui/parser/issue-99625-enum-struct-mutually-exclusive.fixed

```diff
@@ -0,0 +1,15 @@
+// run-rustfix
+
+pub enum Range {
+    //~^ ERROR `enum` and `struct` are mutually exclusive
+    //~^^ ERROR expected identifier, found keyword
+    //~^^^ ERROR expected one of `<`, `where`, or `{`, found `Range`
```

Comment:
These errors are redundant. Fix the comment above and then it should just be one error.